size_t read_long_length_no_check(const BYTE** pp) should be static inline

**Describe the bug**
In lz4.c 1.9.4 at line 1777, this function is not declared static and gets a warning from g++ with -Wmissing-declarations enabled:

```../fast50/lz4/lib/lz4.c:1777:8: error: no previous declaration for ‘size_t read_long_length_no_check(const BYTE**)’ [-Werror=missing-declarations]```

Since this function is only called with lz4.c it should be declared static inline.

**To Reproduce**

g++ -Wmissing-declarations  -Werror lz4.c -o lz4.o

**System (please complete the following information):**
Linux, Ubuntu 18.04
g++ 8.4.0

--- a/lib/lz4.c
+++ b/lib/lz4.c
@@ -1786,7 +1786,7 @@ typedef enum { decode_full_block = 0, partial_decode = 1 } earlyEnd_directive;
  * does not know end of input
  * presumes input is well formed
  * note : will consume at least one byte */
-size_t read_long_length_no_check(const BYTE** pp)
+static size_t read_long_length_no_check(const BYTE** pp)
 {
     size_t b, l = 0;
     do { b = **pp; (*pp)++; l += b; } while (b==255);

